### ISSUE TITLE
question about bn  layer

### ISSUE THREAD

**Issue by oujieww** [XiaLiPKU/EMANet#26]
i am puzzled with the bn layer,in your code ,u did not use torch.nn.batchnorm2d ,What's the difference between the torch.bn with synchronizedbn2d
